book/website/community-handbook/communication-channels.md

```diff
@@ -1,9 +1,88 @@
+- **Fosstodon (@turingway):** Since 2023, we have been using [Fosstodon](https://fosstodon.org/@turingway), a platform for open science practitioners in our community.
+- **LinkedIn Page:** Since late 2023, we have been testing and expanding our presence on [LinkedIn](https://www.linkedin.com/company/the-turing-way) among professional networks.
+
+These social media platforms complement our primary communication channels, providing additional avenues (often requiring manual cross-posting) for community engagement and outreach.
+
+## Asynchronous Communications
+
+In addition to our synchronous communication channels, we leverage various platforms for archiving, sharing, and preserving project-related outputs within _The Turing Way_ community, which serve as asynchronous communication channels.
+
+### Monthly Newsletters
+
+Asynchronous communication is facilitated through {ref}`monthly newsletters<ch-newsletters>`, offering updates on the project's progress, achievements, and upcoming events. 
+These newsletters provide a digestible format for community members to stay informed at their own pace, offering a snapshot of recent developments within _The Turing Way_.
+
+### Zenodo Community Page
+
+_The Turing Way_ community page on [Zenodo](https://zenodo.org/communities/the-turing-way) is the main platform to archive and preserve project-related outputs, including slide decks, reports, datasets (such as the entire GitHub repository and illustrations), and publications. 
+This platform provides persistent identifiers (DOI) to keep stable and citable records, enhancing accessibility and longevity.
+
+### GitHub Repositories
+
+Central to our information storage and sharing, our [GitHub repositories](https://github.com/the-turing-way) serve as the primary hub for project-related documents, resources, and collaborative efforts, offering transparency and collaboration for both synchronous and asynchronous communication.
+
+Since our transition from a single repository to a GitHub organisation, we are moving towards community-maintained repositories on various projects taking place within the community, including governance work, Working Groups, community events and different kinds of communication outputs.
+
+By February 2024, following repositories host specific working group communications and meeting notes:
+- [Accessibility](https://github.com/the-turing-way/accessibility-working-group)
+- [Book Dash](https://github.com/the-turing-way/bookdash)
+- [Infrastructure](https://github.com/the-turing-way/infrastructure-working-group)
+- [Translation and Localisation](https://github.com/TWTranslation)
+- [_The Turing Way_ Community Manager and Project Manager Repo](https://github.com/the-turing-way/2024-rcm-rpm-tasks)
+
+### _The Turing Way_ Book Chapters
```

Comment:
Should the Community Handbook be referenced here?

Reply:
please do add that in.